modules/settings/getCalibrationImage.py

```diff
@@ -0,0 +1,35 @@
+########################################
+#
+# Return img with current calibration mode
+#
+########################################
+
+import cv2
+from classes import frameAnalyzer
+import time
+from classes.capture import capture
+from classes.PiVideoStream import PiVideoStream
+from config import *
+
+
+
+# initialize the camera and grab a reference to the raw camera capture
+camera = PiVideoStream().start()
+# allow the camera to warm up
+time.sleep(2.0)
+
+cap = capture(640, 480)
+# reload config
+config = Config().getConfig()
+
+
+if cap.grab():
+        flag, frame = cap.retrieve()
+
+        # start record
+        capture.captureFrame(frame)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'frame' in unbound method call](https://www.codacy.com/app/saoron_3361/Cardigan/file/2697184446/issues/source?bid=3323288&fileBranchId=3340798#l30)